package.json

```diff
@@ -47,7 +47,7 @@
     "copy-img": "ncp node_modules/patternfly/dist/img src/img",
     "build-css": "sass --style compressed $npm_package_sassIncludes_src $npm_package_sassIncludes_patternfly $npm_package_sassIncludes_patternflyReact $npm_package_sassIncludes_bootstrap $npm_package_sassIncludes_fontAwesome src/",
     "watch-css": "yarn build-css && sass --watch --style compressed $npm_package_sassIncludes_src $npm_package_sassIncludes_patternfly $npm_package_sassIncludes_patternflyReact $npm_package_sassIncludes_bootstrap $npm_package_sassIncludes_fontAwesome src/",
-    "build-js": "react-scripts build",
+    "build-js": "react-scripts --openssl-legacy-provider build",
```

Comment:
other `react-scripts` also need `--openssl-legacy-provider`, like `start:local`

Reply:
Added flag to other `react-scripts` targets too now.

Reply:
@hemslo it seems the `--openssl-legacy-provider` option is not allowed in every `react-scripts` context. 

```
yarn lint && react-scripts --openssl-legacy-provider test --env=jsdom --watchAll=false

...

/usr/bin/node: bad option: --openssl-legacy-provider
```

Will remove it again in other targets, as it seems to run locally w/o that flag too.